### DISCUSSION TITLE
Is `pyrightconfig.json` parsed in VSCode if it's not in the root folder?

### DISCUSSION THREAD

**Discussion by elongl** [microsoft/pyright#6765] [Q&A]
I have a recursive symlink in one of my folders (roughly):
```sh
- vscode_root
  - inner_dir
    - a
      - b
        - a > /root/a
          - b > /root/a/b
...
```
I want to add a `pyrightconfig.json` to `inner_dir` and tell it to exclude `a` directory so that it doesn't enumerate the files endlessly. The question is, would VSCode parse the `pyrightconfig.json` if it's not located in the root of the VSCode folder?

Additionally, might be worth to add a feature that detects recursive symlinks and excludes it because otherwise Pyright doesn't work at all.

**Comment by erictraut** [comment]
The `pyrightconfig.json` file must be in the root directory of your project.

Pyright already has code that detects recursive symlinks when determining which files to analyze. If you are experiencing a problem with this, please file a bug and provide specific repro steps.
